test/functional/http/get/node.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+var node = require('../../../node.js');
+var http = require('../../../common/httpCommunication.js');
+
+describe('GET /api/node/constants', function () {
```

Comment:
I'd create a general describe wrapper `GET /api/node` with two childs `/constans` and `/status`.